stories/components/ProgressCard.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import ProgressCard from '../../components/ProgressCard'
+
+export default {
+  component: ProgressCard,
+  title: 'Components/ProgressCard'
+}
+
+export const Basic: React.FC = () => (
+  <ProgressCard progressCount={0}></ProgressCard>
```

Comment:
If you don't have children it's a common practice to write it with one self closing tag `<ProgressCard progressCount={0} />`

Reply:
Fixed 👍